index.html

```diff
@@ -1,43 +1,264 @@
-        </header>
-        <main>
-            <section class="background">
-        <h1 class="head-title">I’m Mariane <br>
-            Glad to see you!</h1>
-            <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-    <a href="" class="connect"><p>LET’S CONNECT</p></a>
-    <ul class="icons">
-        <li>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins"
+    />
+    <script
+      src="https://kit.fontawesome.com/04970764d6.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+      <div class="wrapper-1">
+    <header class="header">
+      <a href="" class="logo"><p>My Logo</p></a>
+      <nav class="menu"><img src="image/Union.svg" alt="Menu" /></nav>
+    </header>
+    <main>
+          <div class="background">
+        <h1 class="head-title">
+          I’m Mariane <br />
+          Glad to see you!
+        </h1>
+        <p class="content">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <a href="" class="connect"><p>LET’S CONNECT</p></a>
+        <ul class="icons">
+          <li>
             <a href=""><i class="fab fa-twitter"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fab fa-github"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fab fa-linkedin-in"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fab fa-medium-m"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="far fa-hand-peace"></i></a>
+          </li>
+        </ul>
+    </div>
+    </main>
+</div>
+    <!-- Tonic-1 -->
+
+    <div class="wrapper-2">
+      <div class="snapshoot"></div>
+      <h2 class="tonic">Tonic</h2>
+
+      <div class="list">
+        <p class="canopy">CANOPY</p>
+        <a href="" class="date"><p>Back End Dev</p></a>
+        <a href="" class="date"><p>2015</p></a>
+      </div>
+      <p class="content-2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="btns">
+        <li>
+          <p>html</p>
         </li>
         <li>
-            <a href=""><i class="fab fa-github"></i></a>
+          <p>css</p>
         </li>
         <li>
-            <a href=""><i class="fab fa-linkedin-in"></i></a>
+          <p>javascript</p>
         </li>
+      </ul>
+      <a href="#" class="view"><p>See Project</p></a>
+    </div>
+
+    <!-- Multi-Post-Stories-1 -->
+
+<div class="wrapper-3">
```

Comment:
- [ ] Make sure to use `section` tag instead of `div`, which will improve your project to look more semantic and organized, you can read about this in this [link](https://www.w3schools.com/html/html5_semantic_elements.asp)